### PR TITLE
[FIX] fixed LDAP deprecated link

### LINKED ISSUE #8827: Administration: LDAP page link deprecated
### Description:
Hi,

I noticed that the LDAP page in Administration links to a deprecated address in the wiki. The link points to https://rocket.chat/docs/administrator-guides/authentication/ldap/ but the correct address should be https://docs.rocket.chat/administrator-guides/authentication/ldap/

### Server Setup Information:

* Version of Rocket.Chat Server: 0.59.3
* Operating System: Linux
* Deployment Method(snap/docker/tar/etc): Docker
* Number of Running Instances:  1

### Steps to Reproduce:

1. Go to administation
2. Go to LDAP settings
3. Try to open the link in the description

### Expected behavior:

The LDAP wiki page should open

### Actual behavior:

A 404 page pops up

### PR DESCRIPTION
**Pull request by shikhar-scs**
[FIX] fixed page link on administration/LDAP page

@RocketChat/core 

Closes #8827 

The link present on the LDAP page was mentioned as deprecated in the issue mentioned above. And the correct link too was provided in the issue itself. 

Thus, the required changes were made (3 occurrences) in 

Rocket.Chat/packages/rocketchat-i18n/i18n/nl.i18n.json
Rocket.Chat/packages/rocketchat-i18n/i18n/zh-TW.i18n.json
Rocket.Chat/packages/rocketchat-i18n/i18n/en.i18n.json

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/nl.i18n.json [hunk 1]
```diff
@@ -559,14 +559,14 @@
   "Layout_Home_Body": "Home Body",
   "Layout_Home_Title": "Home Titel",
   "Layout_Login_Terms": "Login voorwaarden",
-  "Layout_Privacy_Policy": "Privacybeleid",
+  "Layout_Pnpm install -g eslintrivacy_Policy": "Privacybeleid",
```
Comment (graywolf336) [inline packages/rocketchat-i18n/i18n/nl.i18n.json]: Can you fix this?

**Comment by graywolf336** [comment]
Actually now I'm confused. Why are you changing the links to docs.rocket.chat when that website redirects you to rocket.chat/docs

**Comment by shikhar-scs** [comment]
Well I did mention this on the issue page. It does redirect back, however, in order to provide uniformity and consistency to Rocket Chat I thought the original thing should be there in place. :)

**Comment by shikhar-scs** [comment]
@graywolf336 any updates ?

**Comment by graywolf336** [comment]
@SeanPackham or @rodrigok thoughts on the above?

**Comment by shikhar-scs** [comment]
Are we doing anything about this?

**Comment by graywolf336** [comment]
@shikhar-scs waiting on a response from @SeanPackham or @rodrigok

**Comment by rodrigok** [comment]
The correct address for documentation is `rocket.chat/docs` so the current files are correct, this PR is not necessary anymore (it was for a moment while we tested the `docs.rocket.chat`, but we decided to keep the old format).

So I'll close this PR and the issue linked to it. Let me know if you guys have any comments about that.

Thanks.

**Pull request closed without merging**